docs/Makefile.am

```diff
@@ -537,6 +537,7 @@ OMPI_MAN3 = \
         MPI_Wtick.3 \
         MPI_Wtime.3 \
         MPIX_Query_cuda_support.3 \
+        MPIX_Query_rocm_support.3 \
```

Comment:
I think you forgot to git add this file...?

Reply:
Yep, I just added it :-)

Reply:
if you mean in docs/man-openmpi/man3, the file was actually there, it was just missing from the Makefile.am